modules/contentbox/migrations/2022_10_13_133204_CONTENTBOX-1296-Unique-Site-Slug-Relocation.cfc

```diff
@@ -0,0 +1,26 @@
+component {
+
+	function up( schema, qb ){
+		if ( schema.hasTable( "cb_relocations" ) ) {
```

Comment:
Can you verify this actually works? In my tests, none of the `hasColumn and hasTable()` methods don't he schema builder actually worked.  That's why the migration utilities.cfm has some of these methods.